src/main/java/com/launchcode/AroundTownServer/controllers/UserController.java

```diff
@@ -3,11 +3,8 @@
 import com.launchcode.AroundTownServer.data.UserRepository;
 import com.launchcode.AroundTownServer.models.User;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
```

Comment:
Could we delete this blank line, to keep the imports together in one chunk?

Reply:
I agree, done and pushed